observability/akka-http-metrics/src/test/suite/scala/com/daml/metrics/akkahttp/WebSocketMetricsSpec.scala

```diff
@@ -240,34 +278,47 @@ class AkkaHttpMetricsSpec extends AsyncWordSpec with AkkaBeforeAndAfterAll with
     }
   }
 
-  private val getMessagesReceivedTotalValue: TestMetrics => Long = (metrics: TestMetrics) =>
-    metrics.messagesReceivedTotalValue
+  private val getMessagesReceivedTotalValue: (TestMetrics, Seq[(String, String)]) => Long =
```

Comment:
To improve readability, can we define a case class (e.g., `LabelKeyValue`) for this tuple, please?